txscript/reference_test.go

```diff
@@ -305,15 +305,15 @@ func createSpendingTx(sigScript, pkScript []byte,
 
 	outPoint := wire.NewOutPoint(&chainhash.Hash{}, ^uint32(0))
 	txIn := wire.NewTxIn(outPoint, []byte{OP_0, OP_0})
-	txOut := wire.NewTxOut(outputValue, pkScript)
+	txOut := wire.NewTxOut(outputValue, pkScript, wire.TokenData{})
```

Comment:
Should we have a test that includes token data?

Reply:
Not sure really. The cashtokens test vectors seem to be complete enough. I'm not sure if this would be necessary.